src/components/Hidayah/index.js

```diff
@@ -17,12 +19,50 @@ const PopUpModal = ({ popUpClassName, displayPopUpModal, setDisplayPopUpModal, p
   </div>);
 };
 
+const Levels = ({ level, setLevel }) => (<div>
+  { LEVELS && LEVELS.map((lvl) => (
+    <>
+      <input type="radio" id={lvl} name={lvl} value={lvl} checked={lvl === level} onChange={() => setLevel(lvl)} />
+      <label key={lvl} for={lvl}>{lvl}</label>
+    </>
+  ))}
+</div>);
+
+const SettingsModal = ({ settings, setSettings, hideModal, settingsModalClassName }) => (<div style={{ position: 'absolute', top: 0, left: 0, width: '100%', height: '100%', padding: '1%', backgroundColor: '#ddd' }} className={settingsModalClassName || ''}>
```

Comment:
When you inline the style like you are doing here,  will we be able to still override it?

Reply:
Yes we can override it using `!important`

Reply:
@mohamed7genius no, !important is evil. That is not a sensible way for a client to do iverrides

Reply:
I will change it in a next PR inshaAllah